.storybook/preview.tsx

```diff
@@ -16,93 +17,46 @@ import type { UtagContainer } from '@cmsgov/design-system';
 // Rewire analytics events to log to the console
 (window as UtagContainer).utag = { link: console.log };
 
-const customViewports = {
+const breakpointViewportSizes = {
   extraSmall: {
-    name: 'Extra Small - 320px',
+    name: '$media-width-xs',
     styles: {
       width: '320px',
```

Comment:
Worth pointing out, [Storybook documentation shows height/width values as numbers](https://storybook.js.org/docs/react/essentials/viewport#configuration) and not strings. When you follow through to [the code](https://github.com/storybookjs/storybook/blob/next/code/addons/viewport/src/defaults.ts#L167), it shows they are in fact using strings.